lib/ansible/modules/packaging/os/portage.py

```diff
@@ -172,6 +172,25 @@
     default: None
     version_added: 2.3
 
+  quietbuild:
+    description:
+      - Redirect all build output to logs alone, and do not display it
+      - on stdout (--quiet-build)
+    required: false
+    default: False
+    choices: [ "yes", "no" ]
+    version_added: 2.6
```

Comment:
I'm afraid, this won't make it in 2.6, because it's a bugfix release and  the next one accepting features should be 2.7.

Reply:
Ok I can increase to 2.7, thanks

Reply:
Seems I may need to stick with 2.6 or something since CI is failing on 2.7.

Reply:
It will be allowed to merge this branch when devel will start accepting features for 2.7. It's feature-freeze now. I'm going to guess that might happen once we get 2.6 branch separated.

Reply:
Ok so 2.7 is correct, and I should just ignore the CI failure till next iteration.